common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -339,6 +339,7 @@ class TestCourseImport(ImportTestMixin, StudioCourseTest):
     def page_args(self):
         return [self.browser, self.course_info['org'], self.course_info['number'], self.course_info['run']]
 
+    @flaky  # TNL-6042
     def test_course_updated(self):
```

Comment:
@jcdyer It looks like this problem is more pervasive. See also https://openedx.atlassian.net/browse/TNL-6009?filter=17209.